catalog/4.17/rhcl-operator/catalog.yaml

```diff
@@ -19,6 +19,8 @@ entries:
     replaces: rhcl-operator.v1.1.0
   - name: rhcl-operator.v1.2.0
     replaces: rhcl-operator.v1.1.1
+  - name: rhcl-operator.v1.2.1
```

Comment:
unless I am wrong, we only support latest two OCP GA releases, that should be 4.18 and 4.19 (4.20 to keep that FBC up to date seems valid as well). 

I guess, it should not do any harm releasing in 4.17 as well.